modules/microshift-creating-backups.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-creating-backups_{context}"]
+= Creating backups using the auto-recovery feature
+
+You can use the following procedure to create backups.
+
+[NOTE]
+====
+Creating backups require stopping {microshift-short}, you must determine the best time to stop {microshift-short}.
+====
+
+.Prerequisites
+
+* You have stopped {microshift-short}.
+
+.Procedure
+
+. To store backups, create a backup in the directory you choose by running the following command:
++
+[source,terminal]
+----
+$ sudo microshift backup --auto-recovery _<path_of_directory>_ <1>
```

Comment:
Just an FYI that for some reason, this user replaced value is not showing up as italic in the rendered preview. Instead, it shows the value wrapped in underscores which would be incorrect. I would look into how to fix that, perhaps in the `[source,terminal]` block you need to add something like `subs="quotes"`.

[Here's](https://docs.asciidoctor.org/asciidoc/latest/subs/apply-subs-to-blocks/#set-the-subs-attribute-on-a-block) some reference for the ascii

Reply:
done